src/ReinforcementLearningCore/src/policies/learners.jl

```diff
@@ -7,7 +7,8 @@ abstract type AbstractLearner end
 
 Base.show(io::IO, m::MIME"text/plain", L::AbstractLearner) = show(io, m, convert(AnnotatedStructTree, L))
 
-estimate_reward(L::AbstractLearner, env::AbstractEnv) = env |> state |> send_to_device(L) |> estimate_reward(L) |> send_to_device(env)
+# Take Learner and Environment, get state, send to estimate_reward(Learner, State)
+estimate_reward(L::Le, env::E) where {Le <: AbstractLearner, E <: AbstractEnv} = env |> state |> send_to_device(L.approximator) |> x -> RLCore.estimate_reward(L, x) |> send_to_device(env) 
```

Comment:
Sorry I'm gonna bother you one more time. I hadn't seen it before but `estimate_reward` does not estimate rewards but the (possibly discounted) expected return. So I'd be in favor of renaming this `estimate_return` to avoid confusing future contributors. Additionnaly, estimate_reward might be a name we wish to keep for model-based reinforcement learning as reward prediction is a thing in e.g. Mu-Zero.